src/internal/connector/onedrive/permission.go

```diff
@@ -89,8 +89,9 @@ func getCollectionMetadata(
 // permissions. folderMetas is expected to have all the parent
 // directory metas for this to work.
 func computeParentPermissions(
-	itemPath path.Path,
-	folderMetas map[string]metadata.Metadata,
+	originDir path.Path,
```

Comment:
I don't usually mind renaming things, but please make separate PRs for it. Including name changes with other changes makes it much harder to tell what's an actual change and what's just cleanup